client/app/achievements/achievements.html

```diff
@@ -11,34 +11,11 @@
 
 <div class="container">
   <div class="row text-area">
-    <div class="col-md-8">
```

Comment:
You could also do this using the .col-xs-pull-#  class. https://scotch.io/tutorials/reorder-css-columns-using-bootstrap